packages/svelte/src/internal/server/renderer.js

```diff
@@ -673,3 +716,33 @@ export class SSRState {
 		}
 	}
 }
+
+export class MemoizedUneval {
```

Comment:
Do we really need this? Typically, each hydratable value would be different. And they will hydrate to different objects, even if they do in fact refer to the same object, because we're not unevaling everything at one (and can't, because of custom `encode`/`decode` logic). It seems like overhead